### PR TITLE
Updated various dependencies to resolve security issues

### PR DESCRIPTION
**Pull request by danwom**
### Summary of Changes

_Include a summary of changes then remove this line_

### Tests

_What tests did you add or modify and why? If no tests were added or modified, explain why. Remove this line_

### Documentation

_What documentation did you add or modify and why? Add any relevant links then remove this line_

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [ ] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [ ] PR includes a summary of changes, including screenshots of any UI changes. 
- [ ] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public python functions and the classes in the PR contain docstrings that explain what it does
- [ ] PR passes all tests documented in the [developer guide](https://github.com/lyft/amundsenfrontendlibrary/blob/master/docs/developer_guide.md#testing)

### CODE REVIEW

**Comment by ttannis** [comment]
@danwom Looks like the Jest update caused some issues with our tests. :( Immediate ideas are maybe ts-jest needs to bumped as well, maybe this was a known issue fixed in a later version, or we may have to tweak something in `tsconfig.json` or the `transform` object in `jest.config.js`? Just some shots in the dark here. Let me know if you get stuck for too long.

**Comment by danwom** [comment]
So far I've figured out removing the `import './styles.scss';` statements in our codebase will fix the issue. It's probably related to the `transform` and `jest-css-modules` which was upgraded from `1.1.0 -> 2.1.0`. Will continue investigating

File: amundsen_application/static/jest.config.js [hunk 1]
```diff
@@ -45,6 +44,9 @@ module.exports = {
     'jsx',
     'json',
   ],
+  moduleNameMapper: {
+    '^.+\\.(css|scss)$': '<rootDir>/node_modules/jest-css-modules',
+  },
```
Comment (danwom) [inline amundsen_application/static/jest.config.js]: According to `jest-css-modules` documentation, we're supposed to use `moduleNameMapper`. I'm not sure how or why the rule in `transform` rule was working before.

Comment (ttannis) [reply amundsen_application/static/jest.config.js]: Beats me. This config was the result of reading through various documentation, github issues, and stackoverflow threads. I wonder if its as simple as this being the way to do things with the newer versions. Either way, looks good to me.

**Comment by ttannis** [comment]
Also I haven't looked into why some `mount` still results in issues in the tests yet, but we'll have to figure that out sometime this week -- the inline search results work tests the behavior of `componentDidMount` and `componentWillUnmount`

**Pull request merged**